src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.eligibility.unit.spec.jsx

```diff
@@ -112,10 +112,16 @@ describe('VAOS <VAFacilityPage> eligibility check', () => {
 
     it('should show no clinics message when direct is supported, no clinics available, requests not supported', async () => {
       mockRequestEligibilityCriteria(['983'], []);
+      mockDirectBookingEligibilityCriteria(siteIds, [
+        getDirectBookingEligibilityCriteriaMock({
+          id: '983',
+          typeOfCareId: '211',
+        }),
+      ]);
       mockEligibilityFetches({
         siteId: '983',
         facilityId: '983',
-        typeOfCareId: '323',
+        typeOfCareId: '211',
```

Comment:
Had to change some of these to a different type of care to continue testing the matching clinics check